src/main/bash/drman-githubVCR.sh

```diff
@@ -0,0 +1,26 @@
+# main menu
+
+function __drm_githubVCR() {
+    PS3='Please choose your target VCR: '
+    options=("Create" "Delete" "Other")
+    select opt in "${options[@]}"
+    do
+        case $opt in
+            "Create")
+                echo "Lets Create Github VCR"
+                sh ./create-github-vcr.sh
```

Comment:
@arunprakashpj 

Need to refactor (./create-github-vcr.sh ) out of src/main/bash as all files should begin with drman in this folder will be be picked up by drman_init.sh.

Please do find the refactoring suggestions. 
a) Call a drm function here. This function  could in a helper drm script in  drman-gvcr-helpers.sh
b) move the script (create-github-vcr.sh)  out of src/main/bash to a lib folder under drman , which you can refer in the drm script. however you need to take care of packaging the lib folder and this script.

Reply:
@arunprakashpj  similar suggestion for delete as well.